docs/src/introduction/sway_quickstart.md

```diff
@@ -173,10 +173,15 @@ This starts a Fuel node with a volatile database that will be cleared when shut
 
 To deploy `counter_contract` on your local Fuel node, open a new terminal tab and run the following command from the root of the `wallet_contract` directory:
 
+> **Note**
+> You will require the plugin <a href="https://github.com/FuelLabs/forc-wallet" target="_blank">forc-wallet</a> in order to sign your transactions.
+
 ```sh
-forc deploy --unsigned
+forc deploy --url localhost:4000 --unsigned
```

Comment:
For a local node started with default settings this shouldn't be the case. If that is the case I think we should try to make sure that we can deploy to a local node started without a configuration with just `forc deploy` (i.e lets fix the default url for the `forc-deploy`). 

I will check this on my local to see if it is indeed needed out of the box and edit this comment 👍